cluster/gce/config-common.sh

```diff
@@ -43,9 +40,6 @@ function get-master-size {
   if [[ "$(get-num-nodes)" -gt "500" ]]; then
     suggested_master_size=32
   fi
-  if [[ "$(get-num-nodes)" -gt "2000" ]]; then
```

Comment:
I'm assuming the control plane has been optimised after this change: https://github.com/kubernetes/kubernetes/pull/95947